.github/workflows/linter.yml

```diff
@@ -24,5 +24,4 @@ jobs:
             # in subdirectories. The former only checks sol files in the current directory and directories one level down.
             - name: Run Lint
               run: |
-                  cd contracts/src
-                  npx solhint '**/*.sol' --config ./.solhint.json --ignore-path ./.solhintignore --max-warnings 0
+                  ./scripts/local/lint.sh
```

Comment:
I think we need to either `cd contracts/src` or modify the script to point to `TELEPORTER_PATH/contracts/src/.solhint.json`

Reply:
yup the lint script has `cd $TELEPORTER_PATH/contracts/src`. Lint workflow was failing because of installing locally, and script using `solhint` without npx. Looking online, when installing to use for command line we should install globally. https://nodejs.org/en/blog/npm/npm-1-0-global-vs-local-installation

Reply:
Ah I missed that `cd`. Nice find